src/i18n.js

```diff
@@ -0,0 +1,22 @@
+import i18n from 'i18next'
+import { initReactI18next } from 'react-i18next'
+import LanguageDetector from 'i18next-browser-languagedetector'
+
+import en from './locales/en/translation.json'
+// import de from './locales/de/translation.json'
+
+const resources = {
+  en: {
+    translation: en,
+  },
+  // de: {
+  //   translation: de,
+  // },
+}
```

Comment:
We can also load the appropriate translation file as an asset using `i18next-http-backend`, that would theoretically speed up the process if we ever add a lot of translations.